.circleci/config.yml

```diff
@@ -100,7 +99,6 @@ jobs:
           name: Creating test DB
           command: |
             bundle exec rake db:create db:schema:load --trace
-
```

Comment:
Was this intentionally? Ideally the spacing should remain consistent across the file.

![Screen Shot 2019-04-30 at 11 52 22](https://user-images.githubusercontent.com/1012038/56970854-7107ed00-6b3e-11e9-9c13-ca9a05a3a940.png)

Reply:
Oh weird, definitely not intentional - will fix.